source/_integrations/light.mqtt.markdown

```diff
@@ -8,7 +8,8 @@ ha_iot_class: Configurable
 ha_release: 0.8
 ---
 
-The `mqtt` light platform lets you control your MQTT enabled lights through one of the supported message schemas.
+The `mqtt` light platform lets you control your MQTT enabled lights through one of the supported message schemas, `default`, `json` or `template`.
+Schema is chosen by setting the `schema` configuration parameter.
```

Comment:
Isn't it a bit weird to refer to a configuration option in the introduction text?
A new user hasn't seen that part yet... Maybe just remove this line?
```suggestion
The schema is chosen by setting the `schema` configuration parameter.
```

Reply:
Yeah, line removed.